annotated_test.go

```diff
@@ -1347,6 +1347,78 @@ func TestAnnotate(t *testing.T) {
 		assert.Contains(t, err.Error(), "encountered error while applying annotation using fx.Annotate to go.uber.org/fx_test.TestAnnotate.func1(): cannot apply more than one line of ResultTags")
 	})
 
+	t.Run("invalid ParamTag quotations", func(t *testing.T) {
+		t.Parallel()
+
+		app := NewForTest(t,
+			fx.Provide(
+				// This should just leave newA as it is.
+				fx.Annotate(
+					newA,
+					fx.ParamTags(`name:"something'`),
+				),
+			),
+			fx.Invoke(newB),
+		)
+		err := app.Err()
+		require.Error(t, err)
+		assert.Contains(t, err.Error(), `the ParamTags tag has invalid quotations, did you mean name:"something" ?`)
```

Comment:
Thoughts on using [ErrorContains](https://pkg.go.dev/github.com/stretchr/testify/assert#ErrorContains) instead? It looks like it can combine all 3 of these lines in one.